composer.json

```diff
@@ -36,11 +36,12 @@
     },
     "require-dev": {
         "doctrine/annotations": "~1.0",
-        "phpunit/phpunit": "~4.8.35",
+        "phpunit/phpunit": "~7.4",
```

Comment:
can you also update the minimum php required version to `7.1` please ?

Reply:
woops, forgot that! Thanks :)